src/examples/forms/input-group/InputGroup.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Field, Label, InputGroup, Input } from '@zendeskgarden/react-forms';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => (
+  <Row justifyContent="center">
+    <Col sm={5}>
+      <Field>
+        <Label>Plant</Label>
+        <InputGroup>
+          <Input defaultValue="Soil" />
+          <Button focusInset>Copy</Button>
+        </InputGroup>
```

Comment:
@m-lai I'm still tripping on the examples. What does it mean to "copy soil"? Maybe we can put our heads together to dream up a more fitting nursery-based metaphor :wink:

Reply:
Also, I don't think we'd ever want to use the "Copy" concept with anything but a `readonly` input.

Reply:
@jzempel What about [this example for read-only](https://share.getcloudapp.com/4guJe48Q) where we use "Copy"?